README.md

```diff
@@ -21,6 +26,16 @@ To install dependencies:
     ```bash
     bundle install
     ```
+### Build the site
```

Comment:
This feels unnecessary? jekyll serve will build the site and create the _site folder output, and we've already linked the Jekyll docs.